plugins/outputs/elasticsearch/elasticsearch.go

```diff
@@ -0,0 +1,176 @@
+package elasticsearch
+
+import (
+	"fmt"
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/outputs"
+	"gopkg.in/olivere/elastic.v2"
+	"os"
+	"strings"
+	"time"
+)
+
+type Elasticsearch struct {
+	ServerHost    string
+	IndexName     string
+	EnableSniffer bool
+	Separator     string
+	Client        *elastic.Client
+	Version       string
+}
+
+var sampleConfig = `
+  server_host = "http://10.10.10.10:19200" # required.
+  index_name = "test" # required.
+  enable_sniffer = false
+  delimiter = "_"
+`
+
+type TimeSeries struct {
+	Series []*Metric `json:"series"`
+}
+
+type Metric struct {
+	Metric string   `json:"metric"`
+	Points [1]Point `json:"metrics"`
+}
+
+type Point [2]float64
```

Comment:
Changed in commit:
https://github.com/influxdata/telegraf/pull/1875/commits/862f728635110c159444d2a88bdf3cdf9ae47edc